File: app/src/main/java/com/desgard/sepicat/gua/Feed/FeedActivity.java
package com.desgard.sepicat.gua.Feed;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import com.desgard.sepicat.gua.R;

public class FeedActivity extends AppCompatActivity {
    private String[] data = {"Apple", "Orange", "Banana", "Watermelon"};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_feed);

        ArrayAdapter<String> adapter = new ArrayAdapter<String>(
                FeedActivity.this, android.R.layout.simple_list_item_1, data
        );
        ListView listView = (ListView)findViewById(R.id.feed_list_view);
        listView.setAdapter(adapter);
    }


}

File: app/src/main/java/com/desgard/sepicat/gua/FeedActivity.kt
package com.desgard.sepicat.gua

import android.annotation.SuppressLint
import android.os.Bundle
import android.os.PersistableBundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView

class FeedActivity: AppCompatActivity() {
    @SuppressLint("SetTextI18n")
    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
        val myMessage = TextView(R.layout.myMessage)
        setContentView(myMessage)
    }
}